openpgp/packet/packet.go

```diff
@@ -103,7 +103,7 @@ type partialLengthWriter struct {
 func (w *partialLengthWriter) Write(p []byte) (n int, err error) {
 	bufLen := w.buf.Len()
 	if bufLen > 512 {
-		for power := uint(14); power < 32; power-- {
+		for power := uint(30); power < 32; power-- {
```

Comment:
I think you can remove the condition here; it's a bit redundant
```suggestion
		for power := uint(30); ; power-- {
```